utils/spacewalk-utils.spec

```diff
@@ -114,17 +138,57 @@ spacewalk-python3-pylint $RPM_BUILD_ROOT%{python3_sitelib}
 
 %files
 %defattr(-,root,root)
-%config %{_sysconfdir}/rhn/spacewalk-common-channels.ini
-%config(noreplace) %{_sysconfdir}/rhn/sw-ldap-user-sync.conf
-%attr(755,root,root) %{_bindir}/*
+%doc COPYING.GPLv2 COPYING.GPLv3
+%attr(755,root,root) %{_bindir}/spacewalk-clone-by-date
+%attr(755,root,root) %{_bindir}/spacewalk-manage-channel-lifecycle
+%attr(755,root,root) %{_bindir}/spacewalk-sync-setup
 %dir %{python3_sitelib}/utils
 %{python3_sitelib}/utils/__init__.py*
 %{python3_sitelib}/utils/systemSnapshot.py*
-%{python3_sitelib}/utils/migrateSystemProfile.py*
 %{python3_sitelib}/utils/cloneByDate.py*
 %{python3_sitelib}/utils/depsolver.py*
-%{python3_sitelib}/utils/__pycache__
-%{_mandir}/man8/*
+%dir %{python3_sitelib}/utils/__pycache__
+%{python3_sitelib}/utils/__pycache__/__init__.*
+%{python3_sitelib}/utils/__pycache__/systemSnapshot.*
+%{python3_sitelib}/utils/__pycache__/cloneByDate.*
+%{python3_sitelib}/utils/__pycache__/depsolver.*
+%{_mandir}/man8/spacewalk-clone-by-date.8.gz
+%{_mandir}/man8/spacewalk-sync-setup.8.gz
+
+%files extras
+%defattr(-,root,root)
 %doc COPYING.GPLv2 COPYING.GPLv3
+%attr(755,root,root) %{_bindir}/apply_errata
+%attr(755,root,root) %{_bindir}/delete-old-systems-interactive
+%attr(755,root,root) %{_bindir}/migrate-system-profile
+%attr(755,root,root) %{_bindir}/spacewalk-api
+%attr(755,root,root) %{_bindir}/spacewalk-common-channels
+%attr(755,root,root) %{_bindir}/spacewalk-dump-schema
+%attr(755,root,root) %{_bindir}/spacewalk-export
+%attr(755,root,root) %{_bindir}/spacewalk-export-channels
+%attr(755,root,root) %{_bindir}/spacewalk-final-archive
+%attr(755,root,root) %{_bindir}/spacewalk-hostname-rename
+%attr(755,root,root) %{_bindir}/spacewalk-manage-snapshots
+%attr(755,root,root) %{_bindir}/spacewalk-watch-channel-sync.sh
+%attr(755,root,root) %{_bindir}/sw-ldap-user-sync
+%attr(755,root,root) %{_bindir}/sw-system-snapshot
+%attr(755,root,root) %{_bindir}/taskotop
+%dir %{python3_sitelib}/utils
+%{python3_sitelib}/utils/migrateSystemProfile.py*
+%dir %{python3_sitelib}/utils/__pycache__
```

Comment:
As `-extra` package requires the main package we could remove the two %dir definitions here.
I have the feeling it would be better as it show that we really require the other package to install this one.
Even if we provide the directory, we miss the `__init__.py` which is mandatory to make a python module run.

Reply:
Right. Will give it a try.

Reply:
Fixed. Testing now.